### PR TITLE
fix(ci): re-enable webdriver tests

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

File: scripts/ci.sh [hunk 1]
```diff
@@ -1,39 +1,68 @@
...
+    # eating into its own run time.
+    echo "Sleeping to give webpack-dev-server time to start"
+    sleep 30
+    echo "Completed giving webpack-dev-server time to start"
 
-    # python -m SimpleHTTPServer $TEST_PORT &
-    # pid=$!
+    echo "Starting webdriver"
+    cd ../spot-webdriver
+    npm start || webdriver_exit_code=$?
```
Comment (paweldomas) [inline scripts/ci.sh]: how does this line work:
`npm start || webdriver_exit_code=$?`
?

**Comment by paweldomas** [comment]
The script looks good to me other than the fact that the tests are failing, but I guess you're aware of that.

**Comment by virtuacoplenny** [comment]
The test failures are all related to screensharing. I've added more test reporters to record failures. I'll keep debugging.

**Comment by virtuacoplenny** [comment]
The test failures seem to be an issue where the tests try to screensharing but there is no support for screensharing. This is possibly and http vs https issue. I'm enabling https for the webpack dev server. If this fixes it, I'll make separate commits to address https and some other issues I've encountered.

**Comment by virtuacoplenny** [comment]
jenkins is running an older version of chrome (v. 69) so the tests are failing.

**Comment by virtuacoplenny** [comment]
Should we keep the "https" change? I'm okay with it but I'll have to make a separate commit out of principle. There will be an invalid cert error though.

**Comment by virtuacoplenny** [comment]
Jenkins test this please.

**Comment by paweldomas** [comment]
If it works without the https then lets not keep it. The invalid cert warning is kind of annoying when testing. Does it actually apply to the `npm run start:dev` as well or not? If not then I dont care.

**Comment by virtuacoplenny** [comment]
Okay, I think it's ready now. I have jenkins configured to use the junit results and store the timelime as an archive, and I installed a workspace cleanup plugin so the workspace gets cleared with each run (which will likely increase build times a tick or several).

**Pull request merged**